tests/chart_tests/test_houston_configmap.py

```diff
@@ -514,3 +514,47 @@ def test_houston_configmapwith_update_airflow_runtime_checks_disabled():
     prod = yaml.safe_load(doc["data"]["production.yaml"])
     assert prod["updateAirflowCheckEnabled"] is False
     assert prod["updateRuntimeCheckEnabled"] is False
+
+
+def test_houston_configmap_with_cleanup_airflow_db_enabled():
+    """Validate the houston configmap and its embedded data with
+    cleanupAirflowDb."""
```

Comment:
Can we add a default test case, or else add an assertion to an existing default values test about what the `prod["deployments"]["cleanupAirflowDb"]["enabled"]` should be?

Reply:
done